source/projects/makesite.md

```diff
@@ -0,0 +1,35 @@
+---
+title: makesite.py
+repo: sunainapai/makesite
+homepage: https://github.com/sunainapai/makesite
+language: Python
+license: MIT
+templates: Simple variable substitution
+description: Simple, lightweight, and magic-free static site/blog generator for Python coders.
+startertemplaterepo: sunainapai/makesite
```

Comment:
This needs to be a repo that can be successfully cloned and depoyed as is through Netlify. Should be a starter template repo rather than the project repo.

Reply:
Thank you for the review. I have removed `startertemplaterepo` from the Markdown file and overwritten the commit in this pull request with a single commit that has the correct Markdown file.